### PR TITLE
Add timestamps to log messages

### PR DESCRIPTION
**Pull request by mikaelm12**
Looks like: `[2018-08-09T12:13Z] [Debug] Received message of type PING`
Issue: https://github.com/aspnet/SignalR/issues/2750

### CODE REVIEW

**Comment by mikaelm12** [comment]
Ping 😄

File: clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/ConsoleLogger.java [hunk 1]
```diff
@@ -3,15 +3,23 @@
 
 package com.microsoft.aspnet.signalr;
 
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class ConsoleLogger implements Logger {
     private LogLevel logLevel;
+    private DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
```
Comment (anurse) [inline clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/ConsoleLogger.java]: I think we want to use `Z` without quotes. If we use quotes, it's a literal `Z`, but if we don't use quotes it expands to the time zone in the correct format. So for Redmond you'd get `2018-08-10T12:54-0800`. If we know for sure it's always UTC, we can use `Z`, but I actually think logs should be in local time anyway.

File: clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/ConsoleLogger.java [hunk 2]
```diff
@@ -29,7 +37,8 @@ public void log(LogLevel logLevel, String message) {
     @Override
     public void log(LogLevel logLevel, String formattedMessage, Object... args) {
         if (logLevel.value >= this.logLevel.value) {
-            formattedMessage = formattedMessage + "%n";
+            String timeStamp = dateFormat.format(new Date());
```
Comment (anurse) [inline clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/ConsoleLogger.java]: Is this in local time or UTC?

Comment (mikaelm12) [reply clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/ConsoleLogger.java]: Local time

**Comment by mikaelm12** [comment]
Updated 👍

**Comment by anurse** [comment]
Can you update your example above? With the change you made it should no longer be printing a `Z` at the end (or if it is, we'll need to figure out how to fix that because the `Z` indicates UTC which is inaccurate :))

**Comment by mikaelm12** [comment]
`[2018-08-11T23:00-0700] [Debug] Received message of type PING`

**Comment by anurse** [comment]
> Why is this inaccurate? Also, it's what we did for TS client

Because the TS client renders a UTC time, but the Java client is rendering a local time.

**Pull request merged**